onnxruntime/core/providers/nnapi/nnapi_builtin/builders/model_builder.cc

```diff
@@ -57,10 +57,9 @@ std::vector<std::vector<int>> ModelBuilder::GetSupportedNodes() {
   std::vector<std::vector<int>> supported_node_vecs;
   int32_t android_sdk_ver = GetAndroidSdkVer();
 #ifdef __ANDROID__
-  if (android_sdk_ver < 27) {
-    LOGS_DEFAULT(VERBOSE) << "Android API level "
-                          << android_sdk_ver
-                          << " is lower than 27";
+  if (android_sdk_ver < ORT_NNAPI_MIN_API_LEVEL) {
+    LOGS_DEFAULT(VERBOSE) << "Android API level " << android_sdk_ver
```

Comment:
This is probably worth a warning level log message as it means NNAPI was explicitly enabled but we're ignoring it.